docs/guide/setup/chemical_systems_and_thermodynamic_cycles.rst

```diff
@@ -1,4 +1,42 @@
-ChemicalSystems and thermodynamic cycles
-========================================
+.. _userguide_chemicalsystems_and_components:
 
-issue #759
\ No newline at end of file
+ChemicalSystems, Components and thermodynamic cycles
+====================================================
+
+When thinking about computing estimates in the free energy differences between different states,
+we need an object model to represent the chemistry we are considering.
+In ``openfe`` a :class:`.ChemicalSystem` is used to capture this information,
+and represents the chemical models that are present in each end state.
+
+A ``ChemicalSystem`` **does** include information, where present, on:
+
+* exact protonation state of both the protein and ligand
+* the exact pose of a ligand or conformation of a protein
+* the presence of any crystallographic waters
```

Comment:
Would it also be worth talkinga bout solvation details, e.g. the necessary details to put the system in a box of water?